autoload/importjs.vim

```diff
@@ -52,18 +65,25 @@ function importjs#ExecCommand(command, arg)
     return
   endtry
 
-  let result = json_decode(resultString)
+  if (resultString != "")
+    return importjs#ParseResult(resultString, a:command)
+  endif
+endfunction
+
+function importjs#ParseResult(resultString, command)
+  let result = json_decode(a:resultString)
 
   if (has_key(result, 'error'))
     echoerr result.error
     return
   endif
 
-  if (a:command == "goto" && has_key(result, 'goto'))
+  if ((a:command == "goto" || a:command == "") && has_key(result, 'goto'))
```

Comment:
Yeah, I think it's probably okay to leave that bit out. It was sort of introduced in fac1bbe34e00a457aaaa7ec67dc2c0ab8e2644ce. Before that we just checked the command, after we checked both the command and the result. We just need to check the result.